src/components/Tasks/TaskEdit.tsx

```diff
@@ -92,6 +89,10 @@ class TaskEdit extends React.PureComponent<PropsType> {
       this.state.location = event.location ? event.location : '';
       this.state.description = event.description ? event.description : '';
       this.state.timezone = event.timezone;
+      this.state.priority = event.priority ? event.priority : TaskPriorityType.None;
+
+      // checks if event categories is just the empty string
+      this.state.tags = event.categories && event.categories[0] ? event.categories : [];
```

Comment:
Please enclose the condition in parenthesis, it's very confusing otherwise.
Additionally, if you are just checking if event.categories is an empty string, why not do `event.categories === ''`?

Reply:
Sorry, my comment is poorly written. It should say `checks if event categories contains only the empty string`.
iCAL is kind of silly in how it handles empty lists. If, for instance, you have a task with no tags, its _categories_ property will be saved with the value `['']`.

For backwards compatibility, I first check to see if the task even has a _categories_ property.

This code is a little misleading though. Perhaps it can be written in a cleaner way.

Reply:
See my other comment about the usage of `getValues`, I think it's related. I think it actually shouldn't be getValues, but rather a comma (or is it semicolon?) delimited string that you should deal with on your own. I really don't remember, but I know I've had issues with it before.